lib/players/human.rb

```diff
@@ -0,0 +1,10 @@
+class Player::Human < Player
+  attr_accessor :token
+
+  def move(board)
+    puts "Please enter a move: "
+    input = gets.strip
+    input
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Please enter a move: "
  gets.strip
end
```